docker/test/integration/test_zero_file.py

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the \"License\"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an \"AS IS\" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from minifi import *
+from minifi.test import *
```

Comment:
Import * is not recommended as it's difficult to find the source of the used symbols.

Reply:
I think it's fair to afford a little leeway in the test frameworks ( as we have been ). Aside from the linter. Happy to see a pylint PR if you think that's of importance.

Reply:
I say this because I approved the PRs that introduced this style into the integration test framework -- I think I made comment to this but we discussed that it wasn't a big deal for minifi and minifi.test to import everything.

Reply:
Not a big deal here, not sure about having pyling would be more gain than pain. :)